libkineto/include/ActivityType.h

```diff
@@ -39,13 +40,16 @@ enum class ActivityType {
     HPU_OP, // HPU host side runtime event
     XPU_RUNTIME, // host side xpu runtime events
     COLLECTIVE_COMM, // collective communication
+    PRIVATEUSE1_RUNTIME, // host side privateUse1 runtime events
```

Comment:
Can you add a comment that PRIVATE USE is for custom backends that are not yet released or widely known

Reply:
https://pytorch.org/tutorials/advanced/privateuseone.html

"Privateuse1"  is the recommended way for any new device backend to plugin itself into pytorch despite of whether public released or widely known.

The naming "private use" is indeed a little bit confusing but I guess it's just historical reason.

Reply:
updated